packages/client/hmi-client/src/components/workflow/tera-simulation-workflow.vue

```diff
@@ -24,8 +27,8 @@
 			>
 				<template #body>
 					<tera-model-node
-						v-if="node.operationType === 'ModelOperation' && models"
```

Comment:
Can't we load the model in `onDrop` and pass that in as a prop? It seems kind of "backwards" for a lack of a better word to populate the output first, then load the model based on that.

Reply:
Ya I will try a prop instead, I wasn't sure about going down that path because the assetId could effect other nodes but I think if I just grab the id onMount it could work